src/Security/Authentication/Twitter/src/TwitterHandler.cs

```diff
@@ -259,7 +259,7 @@ private async Task<HttpResponseMessage> ExecuteRequestAsync(string url, HttpMeth
 
             if (formData != null)
             {
-                request.Content = new FormUrlEncodedContent(formData);
+                request.Content = new FormUrlEncodedContent(formData!);
```

Comment:
Why does it need the ❗? It's inside a null-check?

Reply:
FormUrlEncodedContent wants `Dictionary<object?, object?>` and it gets `Dictionary<object, object>`.

Seems like this should just work, but since it doesn't, bang is to make compiler happy.